src/status_im/ui/screens/profile/contact/views.cljs

```diff
@@ -131,7 +133,7 @@
          {:style {:align-self :flex-end
                   :margin-top 16
                   :color      colors/gray}}
-         (str (count @entered-nickname) " / 32")]]
+         (str (count @entered-nickname) " / " nickname-max-length)]]
```

Comment:
Are we sure we are always going to use english numbers?
I mean, Should this be using i18n?

Reply:
We are using this approach to display numbers in various contexts. For instance, we follow a similar pattern in the title input component. Additionally, even in cases where the displayed text includes numbers, we typically pass the actual number value directly from the code. For example: https://github.com/status-im/status-mobile/blob/9479f02b39c0d7b8c53c0b6160f553df77f5acdc/translations/en.json#L2133